### DISCUSSION TITLE
Any way to update Shipping Templates in the API?

### DISCUSSION THREAD

**Discussion by JeffMash** [amzn/selling-partner-api-models#3494] [General]
Right now, we have to manually create a file and upload it through Seller Central to change our Shipping Templates.

Does the API have anything for us to call programmatically to set the Shipping Template for each individual item?

If not, is there any plans for this?   Since we can change just about anything else about a product (price, quantity, lead time, etc), it seems like being able to assign different Shipping Templates would also be beneficial to implement.

**Comment by wszwc** [comment]
Hello, after reading your question, I have a question I would like to ask you. Why is the inventory of the product information I obtained through GetListingItem 0? In fact, I have inventory in the backend, but it seems to be in the delivery template. Have you ever encountered this situation?
